src/main/java/io/ticktok/validator/TicktokValidator.java

```diff
@@ -0,0 +1,14 @@
+package io.ticktok.validator;
+
+import org.apache.commons.lang3.StringUtils;
+
+import static io.ticktok.Ticktok.TicktokException;
+
+public class TicktokValidator {
+
+    public static void validate(String schedule) {
```

Comment:
where do you test this logic?

Reply:
?

Reply:
fixed

Reply:
this is misleading, you call it ticktok validator but you use it on simple string, and you get a schedule but pass it name & schedule
How about you call it NewClockRequest validator and pass it the whole request?